### PR TITLE
[RVM-946] Added GetInstalledRuntimes api

### PR DESCRIPTION
**Pull request by Ben-Drebing**
Added API for user to query the RVM for the versions of all the runtimes installed.
Compatible with RVM 5.3+

fin.system.getInstalledRuntimes( )

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ben-drebing). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

File: src/api/system/system.ts [hunk 1]
```diff
@@ -200,6 +201,13 @@ import { _Window } from '../window/window';
  * @property { string } uuid The uuid of the application
  */
 
+ /**
+ * InstalledRuntimes interface
+ * @typedef { object } InstalledRuntimes
+ * @property { string } action The name of action: "get-installed-runtimes"
```
Comment (datamadic) [inline src/api/system/system.ts]: Is this being returned back to the end users?

Comment (Ben-Drebing) [reply src/api/system/system.ts]: Yes, you can see the tutorial for an example. This API is meant to act in the same way as `getRVMInfo`

File: src/api/system/system.ts [hunk 2]
```diff
@@ -731,6 +739,17 @@ export default class System extends EmitterBase<SystemEvents> {
...
+     * @return {Promise.<InstalledRuntimes>}
+     * @tutorial System.getInstalledRuntimes
+     */
+    // incompatible with standalone node process.
+    public getInstalledRuntimes() : Promise<InstalledRuntimes> {
+        return this.wire.sendAction('get-installed-runtimes')
+            .then(({ payload }) => payload.data);
+    }
+
     /**
```
Comment (licui3936) [inline src/api/system/system.ts]: why return 'action' in response?

Comment (Ben-Drebing) [reply src/api/system/system.ts]: good catch! I updated to just return `payload.data.runtimes` now.

File: src/api/system/system.ts [hunk 3]
```diff
@@ -747,7 +747,7 @@ export default class System extends EmitterBase<SystemEvents> {
     // incompatible with standalone node process.
     public getInstalledRuntimes() : Promise<InstalledRuntimes> {
         return this.wire.sendAction('get-installed-runtimes')
-            .then(({ payload }) => payload.data);
+            .then(({ payload }) => payload.data.runtimes);
```
Comment (licui3936) [inline src/api/system/system.ts]: The response type should be Promise<string[]> if you make this change.

**Pull request merged**